docs/pages/guides/new-architecture.mdx

```diff
@@ -0,0 +1,127 @@
+
+The changes that come with the New Architecture address technical debt of the original implementation and unlock possibilities for solving long standing issues in React Native, such as interoperability with synchronous native APIs (eg: `UITableView`), and pave the way for for [concurrent React support](https://react.dev/blog/2022/03/29/react-v18#what-is-concurrent-react).
+
+## Expo tools and the New Architecture
+
+As of SDK 49, all `expo-*` packages in the [Expo SDK](/versions/latest/) support the New Architecture, with the exception of `expo-updates`. We plan to add support for `expo-updates` in SDK 51.
+
+Additionally, all third-party modules written using the [Expo Modules API](/modules/overview/) support the New Architecture by default! So if you have built your own native modules using this API, no additional work is needed to use them with the New Architecture.
+
+## Initializing a new project with the New Architecture
+
+The easiest way to get started with the New Architecture is to use the `with-new-arch` template when creating a new project:
+
+<Terminal cmd={['$ npx create-expo-app@latest -e with-new-arch']} />
+
+## Enabling the New Architecture in an existing project
+
+> Note: `expo-updates` does not yet support the New Architecture. If you use `expo-updates` in your project, you will need to remove it before continuing.
+
+You can enable the New Architecture in your app using the [expo-build-properties](/versions/latest/sdk/build-properties/) config plugin:
+
+<Step label="1">
+
+Install `expo-build-properties`:
+
+<Terminal cmd={['$ npx expo install expo-build-properties']} />
+
+</Step>
+
+<Step label="2">
+
+Set `newArchEnabled` on target platforms
+
+```json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "ios": {
+            "newArchEnabled": true
+          },
+          "android": {
+            "newArchEnabled": true
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+</Step>
+
+<Step label="3">
+
+Run prebuild and compile your project.
+
+<Terminal cmd={['$ npx expo prebuild --clean', '$ npx expo run:android # or ios', '$ eas build -p android # or ios']} />
+
+</Step>
+
+If the build succeeds, you'll now be running your app with the New Architecture! Depending on the native modules you use, your app may work properly immediately.
+
+Now you can tap around your app and test it out. For most non-trivial apps, you're likely to encounter some issues, such as missing native views that haven't been implemented for the New Architecture yet. Many of the issues you encounter are actionable and can be resolved with some configuration or code changes. We recommend reading the ["Using the compatibility layer"]() and [Troubleshooting](#troubleshooting) sections below for more information.
```

Comment:
Missing link for "Using the compatibility layer" -- but I see that section is still under construction.